src/components/DemoPage/DemoPage.tsx

```diff
@@ -336,6 +345,25 @@ export const DemoPage = () => {
   return (
     <PageLayout title="Component Demo" description="Component demo page intended for showcasing React components.">
       <Flex direction={{ default: 'column' }}>
+        <FlexItem>
+          <Card>
+            <CardBody>
+              <Form>
+                <FormGroup label="Select project name (dynamic search select)">
+                  <SearchSelect
+                    fetchCallback={searchSelectCallback}
+                    attribute="name"
+                    shouldDisplayDescription={true}
+                    onSelect={(value: string | SelectOptionObject) => {
+                      console.log(`DYNAMIC SELECT> ${value}`);
```

Comment:
do we want to log this to console or is it just for demo page?

Reply:
just for the sake of demonstration